shared/common-adapters/text.native.js

```diff
@@ -186,6 +186,7 @@ const sizeGroups: { [key: '40' | '30' | '22' | '18' | '15' | '14']: Object } = {
 
 const textCommon = {
   ...globalStyles.fontRegular,
+  textAlignVertical: 'center',
```

Comment:
this only applies to android and matches what ios does by default. with this off the text labels in the stylesheet headers will be top aligned vs middle for example